### PR TITLE
Frontend customer profile page

### PR DESCRIPTION
**Pull request by lei29**
goto http://localhost:3000/customer/1 and view the page

### CODE REVIEW

File: client/src/components/CuisineList.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const useStyles = makeStyles({
+    name:{
+        backgroundColor:colors.brandLight,
+        color:"white",
+    }
+});
+
+export default function Cuisinelist({ cuisineList }) {
+
+    const classes = useStyles();
```
Comment (smithpg) [inline client/src/components/CuisineList.js]: Does this get used?

File: client/src/components/CuisineList.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    const cuisinecomponent = cuisineList.map((name,i) =>{
```
Comment (smithpg) [inline client/src/components/CuisineList.js]: I suggest changing variable name for clarity
```suggestion
    const cuisineComponents = cuisineList.map((name,i) =>{
```

File: client/src/components/CuisineList.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        return <CuisineTag name={cuisineList[i]}/>;
+    });
+    return (
+        <div>
+            {cuisinecomponent}
```
Comment (smithpg) [inline client/src/components/CuisineList.js]:
```suggestion
            {cuisineComponents}
```

File: client/src/components/GoogleMap.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    center: {
+      lat: position.lat,
+      lng: position.lng
+    },
+    zoom: 13
+  };
+  
+  render() {
+    return (
+      // Important! Always set the container height explicitly
```
Comment (smithpg) [inline client/src/components/GoogleMap.js]:
```suggestion
```

File: client/src/pages/CustomerProfilePage.js [hunk 5]
```diff
@@ -35,25 +35,31 @@ const PageContainer = styled.div`
 
   nav span {
     margin-right: ${layout.spacing(4)};
-    background: white;
+    
   }
 
 `;
+const useStyles = makeStyles({
```
Comment (smithpg) [inline client/src/pages/CustomerProfilePage.js]: If you are applying styles to PageContainer with styled-components, you probably don't need to be using makeStyles here

File: client/src/components/CuisineList.js (hunk 1 above)
Comment (lei29) [reply client/src/components/CuisineList.js]: Yes it has used.

**Pull request merged**